odbc/src/api/data.rs

```diff
@@ -408,6 +408,30 @@ pub mod i16_len {
         *text_length_ptr = len as SmallInt;
         ret
     }
+
+    ///
+    /// set_output_fixed_data writes [`data`], which must be a fixed sized type, to the Pointer [`output_ptr`].
+    /// ODBC drivers assume the output buffer is large enough for fixed types, and are allowed to
+    /// overwrite the buffer if too small a buffer is passed.
+    ///
+    /// # Safety
+    /// This writes to multiple raw C-pointers
+    ///
+    pub unsafe fn set_output_fixed_data<T: core::fmt::Debug>(
```

Comment:
We needed to add this one to the `i16_len` module since `SQLGetInfoW` uses a `*mut SmallInt` for the length.